Formula/macvim.rb

```diff
@@ -18,7 +20,7 @@ class Macvim < Formula
 
   deprecated_option "override-system-vim" => "with-override-system-vim"
 
-  depends_on :xcode => :build
+  depends_on xcode: :build
```

Comment:
Please don't change these for now. We're aware text editors nag to do so, but that's unintentional and we're working on it.